.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,35 @@
+name: 🚀 Feature Request
+description: I have a suggestion!
+labels:
+  - feature
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ## Feature Request
+  - type: checkboxes
+    attributes:
+      label: Prerequisites
+      options:
+        - label: If this is a question/suggestion, I have already considered discuss it on [Discord Server](https://discord.gg/FQ5vgW3)
```

Comment:
Lets change `considered discuss` to `considered to discuss`

Reply:
Superceded by https://github.com/twitch4j/.github/pull/2#discussion_r660202524